libsolidity/ast/Types.h

```diff
@@ -377,6 +377,8 @@ class Type
 	/// Clears all internally cached values (if any).
 	virtual void clearCache() const;
 
+	util::Result<FunctionDefinition const*> operatorDefinition(Token _token, ASTNode const& _scope, bool _unaryOperation) const;
```

Comment:
Does this have a definition?

Reply:
Yes, but it's #12362 :) Thanks for catching it. It shouldn't be here. I removed it.